packages/nuxt/src/module.ts

```diff
@@ -91,9 +91,12 @@ const module: NuxtModule<ModuleOptions> = defineNuxtModule<ModuleOptions>({
       { from: composables, name: 'storeToRefs' },
     ])
 
-    if (!options.storesDirs) {
-      // resolve it against the src dir which is the root by default
-      options.storesDirs = [resolver.resolve(nuxt.options.srcDir, 'stores')]
+    if (options.storesDirs == null) {
+      // Add stores directory for each layer, including the main src dir
+      options.storesDirs = []
+      for (const layer of nuxt.options._layers) {
```

Comment:
Let's use a public API for layers. I don't know what it is but surely there must be a way to resolve the layers from the options, maybe with a nuxt hook?

Reply:
`nuxt.options._layers` seems to be the currently recommended solution for nuxt at present. 
* This solution is introduced in the nuxt documentation. refer: [Multi-Layer Support for Nuxt Modules](https://nuxt.com/docs/guide/going-further/layers#multi-layer-support-for-nuxt-modules).
* In the nuxt source code, this api merging layer is also used. refer: https://github.com/nuxt/nuxt/blob/1c418d0ea3b046f2f7dedc3f39a87cc5be3e90b2/packages/nuxt/src/imports/module.ts#L62